controllers/topics.controller.js

```diff
@@ -1,72 +1,112 @@
-        res.status(200).send({'api' : endpoints})
-    }
-    else {
-        return Promise.reject({status: 404, msg: 'No results Found'})
-    }
+function getApi(req, res, next) {
+  if (endpoints) {
+    res.status(200).send({ api: endpoints });
+  } else {
+    return Promise.reject({ status: 404, msg: 'No results Found' });
+  }
 }
 
-function getArticle(req,res,next){
-    const {article_id} = req.params
-    if (isNaN(article_id)){
-        res.status(400).send({status: 400, msg: 'Bad Request'}) 
-    }
-    findArticleById(article_id)
+function getArticle(req, res, next) {
+  const { article_id } = req.params;
+  if (isNaN(article_id)) {
+    res.status(400).send({ status: 400, msg: 'Bad Request' });
+  }
+  findArticleById(article_id)
     .then((article) => {
-        if (article.length === 1){
-            res.status(200).send({'article':article}) }
-        else {
-            return Promise.reject({status: 404, msg: 'No results Found'})
-        }
+      if (article.length === 1) {
+        res.status(200).send({ article: article });
+      } else {
+        return Promise.reject({ status: 404, msg: 'No results Found' });
+      }
     })
-    .catch((err) => {next(err)})
+    .catch((err) => {
+      next(err);
+    });
 }
 
-function getAllArticles(req,res,next){
-    findAllArticles()
-    .then((articles) => {
-        res.status(200).send({'articles':articles}) }
-    )
+function getAllArticles(req, res, next) {
+  findAllArticles().then((articles) => {
+    res.status(200).send({ articles: articles });
+  });
 }
 
-function getcommentsByArticle(req,res,next){
-    const {article_id} = req.params; 
-    const promises = [checkArticleExists(article_id)] 
-    if (isNaN(article_id)){
-        res.status(400).send({status: 400, msg: 'Bad Request'}) 
-    }
+function getcommentsByArticle(req, res, next) {
+  const { article_id } = req.params;
+  const promises = [checkArticleExists(article_id)];
+  if (isNaN(article_id)) {
+    res.status(400).send({ status: 400, msg: 'Bad Request' });
+  }
 
-    if (article_id){
-        promises.push(findCommentsByArticle(article_id))
-    }
-    //does article_id exist? 
-    Promise.all(promises)
+  if (article_id) {
+    promises.push(findCommentsByArticle(article_id));
+  }
+  Promise.all(promises)
     .then((resolvedPromises) => {
-        const comments = resolvedPromises[1]
-        res.status(200).send({'comments':comments})
+      const comments = resolvedPromises[1];
+      res.status(200).send({ comments: comments });
     })
     .catch((err) => {
-        next(err)
-    })
+      next(err);
+    });
+}
+
+function postComment(req, res, next) {
+  const comment = req.body;
+  if (!comment.hasOwnProperty('username') && !comment.hasOwnProperty('body')) {
+    res.status(400).send({ status: 400, msg: 'Bad Request' });
+  } else {
+    //if statment with existing user check?
+    //exisiting article?
+    const { article_id } = req.params;
+    // console.log(article_id)
```

Comment:
I'm a big fan of both pseudo code and console.logs, however, before we are PRing, we want these to go. Putting in a PR essentially means "I am done with this piece of work"; and the comments and the log wouldn't be things I would expect to see on a finished piece of work